samples/searchAllIamPolicies.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// sample-metadata:
+//   title: Search All Iam Policies
+//   description: Search All Iam Policies.
+//   usage: node searchAllIamPolicies --scope=<SCOPE> --query=<QUERY> --pageSize=<PAGE_SIZE> --pageToken=<PAGE_TOKEN>
+
+async function main() {
+  // [START asset_quickstart_search_all_iam_policies]
+  const util = require('util');
+  const {AssetServiceClient} = require('@google-cloud/asset');
+  const argv = require('yargs').boolean(['autoPaginate']).argv;
```

Comment:
Yargs makes for a great local developer experience, but we try to optimize our samples for copy/paste from cloud.google.com, where we get the vast majority of our usage.  Generally we roll with positional parameters, which doesn't require a dependency, and pass those into the `main` function outside of the tag. 

The rationale, guidance, and examples for specifically how we want to structure these is documented here:
https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head